.github/workflows/review.yml

```diff
@@ -47,6 +48,17 @@ jobs:
           else
             echo "run=false" >> "${GITHUB_OUTPUT}"
           fi
+      - name: Check files for flake8
+        id: flake8
+        run: |
+          if [ "${{ contains(github.event.pull_request.labels.*.name, 'run-ci/flake8') }}" = "true" ]; then
+            echo "run=true" >> "${GITHUB_OUTPUT}"
+          elif git diff --name-only origin/${{ github.base_ref }} HEAD | grep -Eq '.*\.py' ; then
+            echo "run=true" >> "${GITHUB_OUTPUT}"
+            echo 'Python files have changed, need to run flake8.'
```

Comment:
When do we need to check for the "run-ci/flake8" label?

- python files changed - run.
- no changes in python files - don't run.